pkgs/development/libraries/iniparser/default.nix

```diff
@@ -1,51 +1,79 @@
-{ lib, stdenv, fetchFromGitHub }:
+{ lib
```

Comment:
Since the file has been almost completely rewritten, would it make sense to reformat it using `nixfmt-rfc-style` ?

Reply:
It's a backport so I will not do it here. Added the formatting changes in #320233

Reply:
Great!